
<!-- saved from url=(0043)http://stat.ethz.ch/~dettling/bagboost.html -->
<html><head><meta http-equiv="Content-Type" content="text/html; charset=windows-1252">
<title>BagBoosting for Tumor Classification with Gene Expression Data</title>
<meta http-equiv="Content-Language" content="de-ch">

<meta name="author" content="Marcel Dettling">
<meta name="keywords" content="Microarray, Supervised, Clustering,
Classification, Tumor, Expression, Data, Simulation">
<meta http-equiv="expires" content="0">
</head>



<body alink="#cc3232" vlink="#FFFF99" link="FFFF99" bgcolor="#A52A2A" text="WHITE"> 

<p><font size="6"><b>BagBoosting for Tumor Classification with Gene
Expression Data
</b></font></p> <p>&nbsp;</p> 

<table border="0" width="100%">
  <tbody><tr>
    <td width="19%" valign="top">Author :</td> <td width="81%">Marcel
    Dettling<br><br></td>
  </tr>
  <tr>
    <td width="19%" valign="top">Published:</td>
    <td width="81%">In Bioinformatics, December 12th, 2004<br><br></td> 
  </tr>
  <tr>
    <td width="19%" valign="top">Motivation:</td> <td width="81%"> <div align="justify"> Microarray experiments are expected to contribute
    significantly to progress in cancer treatment by enabling a precise and
    early diagnosis. They create a need for class prediction tools that can
    deal with a large number of highly correlated input variables, perform
    feature selection, and provide class probability estimates that serve
    as a quantification of the predictive uncertainty. A very promising
    solution is to combine the two ensemble schemes bagging and boosting to
    a novel algorithm called BagBoosting. <br><br></div></td>
  </tr>
  <tr>
    <td width="19%" valign="top">Results:</td> <td width="81%"> <div align="justify"> When bagging is used as a module in boosting, the
    resulting classifier consistently improves the predictive performance
    and the probability estimates of both bagging and boosting on real and
    simulated gene expression data. This quasi-guaranteed improvement can
    be obtained by simply making a bigger computing effort. The empirical
    advantage is also clearly present when comparing BagBoosting to several
    established class prediction tools for microarray
    data. <br><br></div></td>
  </tr>

  <tr>

    <td width="19%" valign="top">Software:</td> <td width="81%"> <div align="justify"> Software for several modified boosting algorithms
    and for simulation of microarray data is available as an <a href="http://www.r-project.org/">R</a>-Package called <a href="http://stat.ethz.ch/CRAN/src/contrib/Descriptions/boost.html">
    boost</a> from CRAN. There is also a Windows <a href="http://stat.ethz.ch/CRAN/src/contrib/Descriptions/boost.html">
    binary version</a> available. Please report any bugs and pitfalls
    to <a href="mailto:dettling@stat.math.ethz.ch">me</a>.<br><br></div></td>
    </tr>

  <tr>
    <td width="19%" valign="top">Length:</td>
    <td width="81%">11 pages<br><br></td> 
  </tr>
  <tr>
    <td width="19%" valign="top">Reference:</td>
    <td width="81%">Bioinformatics (2004), Vol. 20, No. 18,
    p. 3583-3593.<br><br></td>  
  </tr>
  <tr>
    <td width="19%" valign="top">Download:</td> <td width="81%">The article
    is available online from the Bioinformatics webpage: <a href="http://bioinformatics.oupjournals.org/cgi/content/abstract/bth447?ijkey=11C8Zp5bT5EFY&keytype=ref">click
    here</a>. A slightly outdated preprint is available as <a href="ftp://stat.ethz.ch/Manuscripts/dettling/bagboost.pdf">PDF</a>(334k)
    and <a href="ftp://stat.ethz.ch/Manuscripts/dettling/bagboost.ps">PS</a>(327k).
    For reprint requests and further information, please contact me via <a href="mailto:dettling@stat.math.ethz.ch">e-mail</a>.
    <br><br></td>
  </tr>
  <tr>
    <td width="19%" valign="top">Datasets:</td> <td width="81%"> <div align="justify">For the purpose of comparison, you can download the
    preprocessed microarray gene expression datasets exactly as I used them
    in my empirical study. They are provided as R data files and contain
    both the expression matrix and the response variable. Click here for
    the <a href="ftp://stat.ethz.ch/Manuscripts/dettling/leukemia.rda">Leukemia</a>
    data (2010k), <a href="ftp://stat.ethz.ch/Manuscripts/dettling/colon.rda">Colon</a> data
    (970k), <a href="ftp://stat.ethz.ch/Manuscripts/dettling/prostate.rda">Prostate</a>
    data (4809k), <a href="ftp://stat.ethz.ch/Manuscripts/dettling/lymphoma.rda">Lymphoma</a>
    data (1951k), <a href="ftp://stat.ethz.ch/Manuscripts/dettling/srbct.rda">SRBCT</a> data
    (1137k) and <a href="ftp://stat.ethz.ch/Manuscripts/dettling/brain.rda">Brain</a> data
    (1837k). For information about the origin and the preprocessing of the
    datasets, please read my paper about <a href="http://stat.ethz.ch/~dettling/supercluster.html">Supervised Clustering of Genes</a>.
    <br><br></div></td>
  </tr>
  <tr>
    <td width="19%" valign="top">Related material:</td> <td width="81%"> <div align="justify">Our first paper about boosting for tumor classification
    with gene expression data is available <a href="http://stat.ethz.ch/~dettling/boosting.html">here</a>.
  </div></td></tr>
</tbody></table>

<br>
<br>
<br>
<hr>
<font size="2">
<table border="0" width="100%">
  <tbody><tr>
    <td width="50%">
    <a href="http://stat.ethz.ch/~dettling/publications.html">Back</a> / <a href="http://stat.ethz.ch/~dettling/index.html">Home</a>
    </td>
    <td width="50%" align="right">
    <a href="mailto:dettling@stat.math.ethz.ch">Marcel Dettling</a>,
    20.04.2005
    </td>
  </tr>
</tbody></table>

<!-- Start of StatCounter Code -->
<script type="text/javascript" language="javascript">
var sc_project=663864; 
var sc_partition=5; 
var sc_security="0c46e399"; 
var sc_invisible=1; 
</script>

<script type="text/javascript" language="javascript" src="./Datasets_BagBoosting for Tumor Classification with Gene Expression Data_files/counter.js"></script><noscript>&lt;a href="http://www.statcounter.com/" target="_blank"&gt;&lt;img  src="http://c6.statcounter.com/counter.php?sc_project=663864&amp;amp;java=0&amp;amp;security=0c46e399" alt="free statistics" border="0"&gt;&lt;/a&gt; </noscript>
<!-- End of StatCounter Code -->




</font></body></html>